pkg/reconciler/instances/istio/actions/performer.go

```diff
@@ -252,6 +254,11 @@ func (c *DefaultIstioPerformer) Update(context context.Context, kubeConfig, isti
 		return err
 	}
 
+	ingressGatewayNeedsRestart, err := merge.NeedsIngressGatewayRestart(context, c.provider, kubeConfig, logger)
```

Comment:
I'd say that in golang short variable names are preferred, and from the context it's clear what needs to be restarted. How about "restartRequired"?